datapipelines/src/main/kotlin/io/customer/datapipelines/migration/TrackingMigrationProcessor.kt

```diff
@@ -45,6 +46,11 @@ internal class TrackingMigrationProcessor(
         dataPipelineInstance.identify(userId = identifier)
     }
 
+    override fun processDeviceMigration(oldDeviceIdentifier: String) = runCatching {
```

Comment:
We should fix this warning before merging

```
The corresponding parameter in the supertype 'MigrationProcessor' is named 'token'. This may cause problems when calling this function with named arguments
```